# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/EventStateManager.cpp
# Commit: ab54092b8afc
# Full Hash: ab54092b8afc8e56dd3f8162344a4ce38ec45999
# Author: Niklas Goegge <ngogge@mozilla.com>
# Date: 2021-04-29 09:26:05
# Regressor Bug: 1708150
# File Overlap Count: 1
# Description:
#   Bug 1708150 - Add user activation flag to reload, goBack and goForward r=ckerschb,Gijs,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D110245
# ==============================================================================

diff -r a9a52f01de13 -r ab54092b8afc dom/events/EventStateManager.cpp
--- a/dom/events/EventStateManager.cpp	Wed Apr 28 11:09:38 2021 +0000
+++ b/dom/events/EventStateManager.cpp	Wed Apr 28 11:26:49 2021 +0000
@@ -2362,11 +2362,11 @@
       // This is doing user-initiated history traversal, hence we want
       // to require that history entries we navigate to have user interaction.
       if (direction > 0)
-        webNav->GoBack(
-            StaticPrefs::browser_navigation_requireUserInteraction());
+        webNav->GoBack(StaticPrefs::browser_navigation_requireUserInteraction(),
+                       true);
       else
         webNav->GoForward(
-            StaticPrefs::browser_navigation_requireUserInteraction());
+            StaticPrefs::browser_navigation_requireUserInteraction(), true);
     }
   }
 }